elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -14,7 +13,13 @@
             {{^hide_letter_keys}}<div class="pl-multiple-choice-key-label">({{key}})</div>{{/hide_letter_keys}}
             <div class="ml-1 mr-1">{{{html}}}</div>
         </label>
-            
+
+        {{#display_feedback}}
+        <div class="pl-multiple-choice-feedback">
+            {{{feedback}}}
+        </div>
+        {{/display_feedback}}
```

Comment:
I love the display of feedback in the question panel next to the selected item. But in the case where the student makes multiple submissions, the feedback will only be visible for the most recent submission (in the question panel). Can we additionally display the feedback in the submission panel, next to the submitted answer? That way the student will be able to see the history of their feedback together with their history of submitted answers.

Reply:
Just implemented this, here's what it looks like:
<img width="679" alt="Screen Shot 2021-08-30 at 8 49 01 PM" src="https://user-images.githubusercontent.com/68084820/131438762-dbeeb8da-0814-4096-b218-9d4d2620264e.png">

Reply:
Perfect!